### PR TITLE
Adds a new crutch mechanic

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

You can now hide tiny items inside crutches by screwing their tips.
## Why It's Good For The Game

image

Considering that crutches are BULKY one could justify the inclusion of this obscure mechanic, allowing people to hide their illegals by pretending to be crippled.

## Testing

- Did screw and unscrew the crutches multiple time
- Tried to insert items with different sizes
- Deleted it to make sure its dropping the hidden item
## Changelog
:cl:
tweak: You can now hide tiny items inside crutches by screwing their tips
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/misc_items.dm [hunk 1]
```diff
@@ -51,15 +51,55 @@
...
+
+/obj/item/crutches/update_icon_state() //Currently only here to fuck with the on-mob icons.
 	icon_state = "crutches0"
 	return ..()
 
+/obj/item/crutches/attackby(obj/item/I, mob/user, params)
+	. = ..()
+	if(!is_open)
+		return
+	if(!hidden && !isscrewdriver(I) && I.w_class == WEIGHT_CLASS_TINY)
```
Comment (Contrabang) [inline code/game/objects/items/weapons/misc_items.dm:77]: Please use tool behavior instead, it should look something like this
```suggestion
	if(!hidden && I.tool_behavior != TOOL_SCREWDRIVER && I.w_class == WEIGHT_CLASS_TINY)
```

Comment (Henri215) [reply code/game/objects/items/weapons/misc_items.dm:77]: Added in next commit, thanks.

**Comment by ArcticValor** [comment]
ehe, reminds me of that clip where an officer takes off someone's prosthetic leg and pulls out a plug only to dump out tons of cocaine!

**Comment by Christasmurf** [comment]
> ehe, reminds me of that clip where an officer takes off someone's prosthetic leg and pulls out a plug only to dump out tons of cocaine!

https://www.youtube.com/watch?v=OF8TSHN-JDA&ab_channel=QuoteTheGuy

**Comment by Henri215** [comment]
> > ehe, reminds me of that clip where an officer takes off someone's prosthetic leg and pulls out a plug only to dump out tons of cocaine!
> 
> https://www.youtube.com/watch?v=OF8TSHN-JDA&ab_channel=QuoteTheGuy

I was about to ask for the clip over discord :D

**Review by lewcc — APPROVED** [review]

**Pull request merged**